File: src/test/java/com/polykhel/billies/domain/WalletTest.java
package com.polykhel.billies.domain;

import static org.assertj.core.api.Assertions.assertThat;

import com.polykhel.billies.web.rest.TestUtil;
import org.junit.jupiter.api.Test;

class WalletTest {

    @Test
    void equalsVerifier() throws Exception {
        TestUtil.equalsVerifier(Wallet.class);
        Wallet wallet1 = new Wallet();
        wallet1.setId(1L);
        Wallet wallet2 = new Wallet();
        wallet2.setId(wallet1.getId());
        assertThat(wallet1).isEqualTo(wallet2);
        wallet2.setId(2L);
        assertThat(wallet1).isNotEqualTo(wallet2);
        wallet1.setId(null);
        assertThat(wallet1).isNotEqualTo(wallet2);
    }
}

File: src/test/kotlin/com/polykhel/billnext/domain/WalletTest.kt
package com.polykhel.billnext.domain

import com.polykhel.billnext.web.rest.equalsVerifier
import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class WalletTest {

    @Test
    fun equalsVerifier() {
        equalsVerifier(Wallet::class)
        val wallet1 = Wallet()
        wallet1.id = 1L
        val wallet2 = Wallet()
        wallet2.id = wallet1.id
        assertThat(wallet1).isEqualTo(wallet2)
        wallet2.id = 2L
        assertThat(wallet1).isNotEqualTo(wallet2)
        wallet1.id = null
        assertThat(wallet1).isNotEqualTo(wallet2)
    }
}
